app.py

```diff
@@ -167,6 +168,12 @@ def edit_patient(folder_hash):
 
     return render_template('patient_bio_info_edit.html', form=form)
 
+@app.route('/add_data/<folder_hash>')
```

Comment:
because patient has to first be registered with patient_info. then only folder number is carried over to other forms. That is why I need to be able to take the patient.folder_number to biopsy.folder_number.